mopidy/http/handlers.py

```diff
@@ -143,16 +148,31 @@ def set_mopidy_headers(request_handler):
         'X-Mopidy-Version', mopidy.__version__.encode('utf-8'))
 
 
+def check_origin(origin, request_headers, origin_whitelist):
+    if origin is None:
+        logger.debug('Origin was not set')
+        return False
+    origin_whitelist.add(request_headers.get('Host', None))
```

Comment:
This allows access for a webpage at `foo:80` if the RPC interface also is available at `foo:80`? Cute. That voids my comment about `localhost:6600` as the default config value.